pom.xml

```diff
@@ -170,5 +163,89 @@
                 </repository>
             </distributionManagement>
         </profile>
+        <profile>
```

Comment:
Java really did go all in on xml 😭

Reply:
Yeah, next one we can try Gradle, not so much xml